src/ontology/edits/oeo-physical.omn

```diff
@@ -718,7 +718,7 @@ Biofuels can be derived directly from plants (i.e. energy crops), or indirectly
 Class: Biogas
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A gas composed principally of methane and carbon dioxide produced by anaerobic digestion of biomass.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Biogas is a portion of matter which has a gaseous state and is composed principally of methane and carbon dioxide produced by anaerobic digestion of biomass.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Biogas is a gaseous biofuel and is composed principally of methane and carbon dioxide produced by anaerobic digestion of biomass.",
```

Reply:
Same as charcoal, biogas is primarily a subclass of portion of matter. Would it be enough to add another sentence like "It is used as a fuel" at the end?

Reply:
Actually, for biogas it could make sense to make it a direct subclass of biofuel. I think the term is primarily used in the fuel context, unlike e.g. methane.

Reply:
ok it is now a direct subclass of Biofuel with the corresponding definition